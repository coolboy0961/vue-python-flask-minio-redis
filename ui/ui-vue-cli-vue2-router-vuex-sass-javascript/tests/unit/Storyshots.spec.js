import initStoryshots from "@storybook/addon-storyshots";
import { imageSnapshot } from "@storybook/addon-storyshots-puppeteer";

jest.spyOn(console, "error").mockImplementation(() => {});
initStoryshots({
  suite: "Image storyshots",
  test: imageSnapshot({
    storybookUrl: "http://localhost:6006/", // 如果你的 Storybook 运行在其他端口上，请进行相应修改
    // 其他可选配置
  }),
});
