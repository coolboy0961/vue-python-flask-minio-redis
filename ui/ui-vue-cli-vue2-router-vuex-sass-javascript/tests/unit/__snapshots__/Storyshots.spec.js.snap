// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Component/AboutView Default 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <div
      class="about"
    >
      <h1>
        This is an about page
      </h1>
       
      <br />
       
      <h1
        data-testid="count-text"
        id="count-text"
      >
        count: 0
      </h1>
       
      <button
        data-testid="increment-button"
        id="increment-button"
      >
        
    Count
  
      </button>
    </div>
  </v-container>
</v-app>
`;

exports[`Storyshots Component/HelloWorld Test Message 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <v-container>
      <v-row
        class="text-center"
      >
        <h1>
          Test Message
        </h1>
         
        <v-col
          cols="12"
        >
          <v-img
            class="my-3"
            contain=""
            height="200"
            src=""
          />
        </v-col>
         
        <v-col
          class="mb-4"
        >
          <h1
            class="display-2 font-weight-bold mb-3"
          >
            Welcome to Vuetify
          </h1>
           
          <p
            class="subheading font-weight-regular"
          >
            
        For help and collaboration with other Vuetify developers,
        
            <br />
            please join our online
        
            <a
              href="https://community.vuetifyjs.com"
              target="_blank"
            >
              Discord Community
            </a>
          </p>
        </v-col>
         
        <v-col
          class="mb-5"
          cols="12"
        >
          <h2
            class="headline font-weight-bold mb-3"
          >
            What's next?
          </h2>
           
          <v-row
            justify="center"
          >
            <a
              class="subheading mx-3"
              href="https://vuetifyjs.com/components/api-explorer"
              target="_blank"
            >
              
          Explore components
        
            </a>
            <a
              class="subheading mx-3"
              href="https://vuetifyjs.com/getting-started/pre-made-layouts"
              target="_blank"
            >
              
          Select a layout
        
            </a>
            <a
              class="subheading mx-3"
              href="https://vuetifyjs.com/getting-started/frequently-asked-questions"
              target="_blank"
            >
              
          Frequently Asked Questions
        
            </a>
          </v-row>
        </v-col>
         
        <v-col
          class="mb-5"
          cols="12"
        >
          <h2
            class="headline font-weight-bold mb-3"
          >
            Important Links
          </h2>
           
          <v-row
            justify="center"
          >
            <a
              class="subheading mx-3"
              href="https://vuetifyjs.com"
              target="_blank"
            >
              
          Documentation
        
            </a>
            <a
              class="subheading mx-3"
              href="https://community.vuetifyjs.com"
              target="_blank"
            >
              
          Chat
        
            </a>
            <a
              class="subheading mx-3"
              href="https://madewithvuejs.com/vuetify"
              target="_blank"
            >
              
          Made with Vuetify
        
            </a>
            <a
              class="subheading mx-3"
              href="https://twitter.com/vuetifyjs"
              target="_blank"
            >
              
          Twitter
        
            </a>
            <a
              class="subheading mx-3"
              href="https://medium.com/vuetify"
              target="_blank"
            >
              
          Articles
        
            </a>
          </v-row>
        </v-col>
         
        <v-col
          class="mb-5"
          cols="12"
        >
          <h2
            class="headline font-weight-bold mb-3"
          >
            Ecosystem
          </h2>
           
          <v-row
            justify="center"
          >
            <a
              class="subheading mx-3"
              href="https://github.com/vuetifyjs/vuetify-loader"
              target="_blank"
            >
              
          vuetify-loader
        
            </a>
            <a
              class="subheading mx-3"
              href="https://github.com/vuetifyjs/vuetify"
              target="_blank"
            >
              
          github
        
            </a>
            <a
              class="subheading mx-3"
              href="https://github.com/vuetifyjs/awesome-vuetify"
              target="_blank"
            >
              
          awesome-vuetify
        
            </a>
          </v-row>
        </v-col>
         
        <v-col
          class="mb-5"
          cols="12"
        >
          <h2>
            Call API when page mounted:
          </h2>
           
          <v-row
            justify="center"
          >
            <p>
              {}
            </p>
          </v-row>
        </v-col>
      </v-row>
    </v-container>
  </v-container>
</v-app>
`;

exports[`Storyshots Example/Button Large 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <button
      class="storybook-button storybook-button--secondary storybook-button--large"
      type="button"
    >
      
  Button

    </button>
  </v-container>
</v-app>
`;

exports[`Storyshots Example/Button Primary 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <button
      class="storybook-button storybook-button--primary storybook-button--medium"
      type="button"
    >
      
  Button

    </button>
  </v-container>
</v-app>
`;

exports[`Storyshots Example/Button Secondary 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <button
      class="storybook-button storybook-button--secondary storybook-button--medium"
      type="button"
    >
      
  Button

    </button>
  </v-container>
</v-app>
`;

exports[`Storyshots Example/Button Small 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <button
      class="storybook-button storybook-button--secondary storybook-button--small"
      type="button"
    >
      
  Button

    </button>
  </v-container>
</v-app>
`;

exports[`Storyshots Example/Header Logged In 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <header>
      <div
        class="wrapper"
      >
        <div>
          <svg
            height="32"
            viewBox="0 0 32 32"
            width="32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="none"
              fill-rule="evenodd"
            >
              <path
                d="M10 0h12a10 10 0 0110 10v12a10 10 0 01-10 10H10A10 10 0 010 22V10A10 10 0 0110 0z"
                fill="#FFF"
              />
               
              <path
                d="M5.3 10.6l10.4 6v11.1l-10.4-6v-11zm11.4-6.2l9.7 5.5-9.7 5.6V4.4z"
                fill="#555AB9"
              />
               
              <path
                d="M27.2 10.6v11.2l-10.5 6V16.5l10.5-6zM15.7 4.4v11L6 10l9.7-5.5z"
                fill="#91BAF8"
              />
            </g>
          </svg>
           
          <h1>
            Acme
          </h1>
        </div>
         
        <div>
          <span
            class="welcome"
          >
            Welcome, 
            <b>
              Jane Doe
            </b>
            !
          </span>
           
          <button
            class="storybook-button storybook-button--secondary storybook-button--small"
            type="button"
          >
            
  Log out

          </button>
           
          <!---->
           
          <!---->
        </div>
      </div>
    </header>
  </v-container>
</v-app>
`;

exports[`Storyshots Example/Header Logged Out 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <header>
      <div
        class="wrapper"
      >
        <div>
          <svg
            height="32"
            viewBox="0 0 32 32"
            width="32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="none"
              fill-rule="evenodd"
            >
              <path
                d="M10 0h12a10 10 0 0110 10v12a10 10 0 01-10 10H10A10 10 0 010 22V10A10 10 0 0110 0z"
                fill="#FFF"
              />
               
              <path
                d="M5.3 10.6l10.4 6v11.1l-10.4-6v-11zm11.4-6.2l9.7 5.5-9.7 5.6V4.4z"
                fill="#555AB9"
              />
               
              <path
                d="M27.2 10.6v11.2l-10.5 6V16.5l10.5-6zM15.7 4.4v11L6 10l9.7-5.5z"
                fill="#91BAF8"
              />
            </g>
          </svg>
           
          <h1>
            Acme
          </h1>
        </div>
         
        <div>
          <!---->
           
          <!---->
           
          <button
            class="storybook-button storybook-button--secondary storybook-button--small"
            type="button"
          >
            
  Log in

          </button>
           
          <button
            class="storybook-button storybook-button--primary storybook-button--small"
            type="button"
          >
            
  Sign up

          </button>
        </div>
      </div>
    </header>
  </v-container>
</v-app>
`;

exports[`Storyshots Example/Page Logged In 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <article>
      <header>
        <div
          class="wrapper"
        >
          <div>
            <svg
              height="32"
              viewBox="0 0 32 32"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="none"
                fill-rule="evenodd"
              >
                <path
                  d="M10 0h12a10 10 0 0110 10v12a10 10 0 01-10 10H10A10 10 0 010 22V10A10 10 0 0110 0z"
                  fill="#FFF"
                />
                 
                <path
                  d="M5.3 10.6l10.4 6v11.1l-10.4-6v-11zm11.4-6.2l9.7 5.5-9.7 5.6V4.4z"
                  fill="#555AB9"
                />
                 
                <path
                  d="M27.2 10.6v11.2l-10.5 6V16.5l10.5-6zM15.7 4.4v11L6 10l9.7-5.5z"
                  fill="#91BAF8"
                />
              </g>
            </svg>
             
            <h1>
              Acme
            </h1>
          </div>
           
          <div>
            <!---->
             
            <!---->
             
            <button
              class="storybook-button storybook-button--secondary storybook-button--small"
              type="button"
            >
              
  Log in

            </button>
             
            <button
              class="storybook-button storybook-button--primary storybook-button--small"
              type="button"
            >
              
  Sign up

            </button>
          </div>
        </div>
      </header>
       
      <section>
        <h2>
          Pages in Storybook
        </h2>
         
        <p>
          
      We recommend building UIs with a
      
          <a
            href="https://componentdriven.org"
            rel="noopener noreferrer"
            target="_blank"
          >
            <strong>
              component-driven
            </strong>
          </a>
          
      process starting with atomic components and ending with pages.
    
        </p>
         
        <p>
          
      Render pages with mock data. This makes it easy to build and review page
      states without needing to navigate to them in your app. Here are some
      handy patterns for managing page data in Storybook:
    
        </p>
         
        <ul>
          <li>
            
        Use a higher-level connected component. Storybook helps you compose
        such data from the "args" of child component stories
      
          </li>
           
          <li>
            
        Assemble data in the page component from your services. You can mock
        these services out using Storybook.
      
          </li>
        </ul>
         
        <p>
          
      Get a guided tutorial on component-driven development at
      
          <a
            href="https://storybook.js.org/tutorials/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Storybook tutorials
          </a>
          
      . Read more in the
      
          <a
            href="https://storybook.js.org/docs"
            rel="noopener noreferrer"
            target="_blank"
          >
            docs
          </a>
          
      .
    
        </p>
         
        <div
          class="tip-wrapper"
        >
          <span
            class="tip"
          >
            Tip
          </span>
          
      Adjust the width of the canvas with the
      
          <svg
            height="10"
            viewBox="0 0 12 12"
            width="10"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="none"
              fill-rule="evenodd"
            >
              <path
                d="M1.5 5.2h4.8c.3 0 .5.2.5.4v5.1c-.1.2-.3.3-.4.3H1.4a.5.5 0 01-.5-.4V5.7c0-.3.2-.5.5-.5zm0-2.1h6.9c.3 0 .5.2.5.4v7a.5.5 0 01-1 0V4H1.5a.5.5 0 010-1zm0-2.1h9c.3 0 .5.2.5.4v9.1a.5.5 0 01-1 0V2H1.5a.5.5 0 010-1zm4.3 5.2H2V10h3.8V6.2z"
                fill="#999"
                id="a"
              />
            </g>
          </svg>
          
      Viewports addon in the toolbar
    
        </div>
      </section>
    </article>
  </v-container>
</v-app>
`;

exports[`Storyshots Example/Page Logged Out 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <article>
      <header>
        <div
          class="wrapper"
        >
          <div>
            <svg
              height="32"
              viewBox="0 0 32 32"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="none"
                fill-rule="evenodd"
              >
                <path
                  d="M10 0h12a10 10 0 0110 10v12a10 10 0 01-10 10H10A10 10 0 010 22V10A10 10 0 0110 0z"
                  fill="#FFF"
                />
                 
                <path
                  d="M5.3 10.6l10.4 6v11.1l-10.4-6v-11zm11.4-6.2l9.7 5.5-9.7 5.6V4.4z"
                  fill="#555AB9"
                />
                 
                <path
                  d="M27.2 10.6v11.2l-10.5 6V16.5l10.5-6zM15.7 4.4v11L6 10l9.7-5.5z"
                  fill="#91BAF8"
                />
              </g>
            </svg>
             
            <h1>
              Acme
            </h1>
          </div>
           
          <div>
            <!---->
             
            <!---->
             
            <button
              class="storybook-button storybook-button--secondary storybook-button--small"
              type="button"
            >
              
  Log in

            </button>
             
            <button
              class="storybook-button storybook-button--primary storybook-button--small"
              type="button"
            >
              
  Sign up

            </button>
          </div>
        </div>
      </header>
       
      <section>
        <h2>
          Pages in Storybook
        </h2>
         
        <p>
          
      We recommend building UIs with a
      
          <a
            href="https://componentdriven.org"
            rel="noopener noreferrer"
            target="_blank"
          >
            <strong>
              component-driven
            </strong>
          </a>
          
      process starting with atomic components and ending with pages.
    
        </p>
         
        <p>
          
      Render pages with mock data. This makes it easy to build and review page
      states without needing to navigate to them in your app. Here are some
      handy patterns for managing page data in Storybook:
    
        </p>
         
        <ul>
          <li>
            
        Use a higher-level connected component. Storybook helps you compose
        such data from the "args" of child component stories
      
          </li>
           
          <li>
            
        Assemble data in the page component from your services. You can mock
        these services out using Storybook.
      
          </li>
        </ul>
         
        <p>
          
      Get a guided tutorial on component-driven development at
      
          <a
            href="https://storybook.js.org/tutorials/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Storybook tutorials
          </a>
          
      . Read more in the
      
          <a
            href="https://storybook.js.org/docs"
            rel="noopener noreferrer"
            target="_blank"
          >
            docs
          </a>
          
      .
    
        </p>
         
        <div
          class="tip-wrapper"
        >
          <span
            class="tip"
          >
            Tip
          </span>
          
      Adjust the width of the canvas with the
      
          <svg
            height="10"
            viewBox="0 0 12 12"
            width="10"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="none"
              fill-rule="evenodd"
            >
              <path
                d="M1.5 5.2h4.8c.3 0 .5.2.5.4v5.1c-.1.2-.3.3-.4.3H1.4a.5.5 0 01-.5-.4V5.7c0-.3.2-.5.5-.5zm0-2.1h6.9c.3 0 .5.2.5.4v7a.5.5 0 01-1 0V4H1.5a.5.5 0 010-1zm0-2.1h9c.3 0 .5.2.5.4v9.1a.5.5 0 01-1 0V2H1.5a.5.5 0 010-1zm4.3 5.2H2V10h3.8V6.2z"
                fill="#999"
                id="a"
              />
            </g>
          </svg>
          
      Viewports addon in the toolbar
    
        </div>
      </section>
    </article>
  </v-container>
</v-app>
`;

exports[`Storyshots Pages/App Default 1`] = `
<v-app>
  <v-container
    fluid=""
  >
    <v-app>
      <v-app-bar
        app=""
        color="primary"
        dark=""
      >
        <div
          class="d-flex align-center"
        >
          <v-img
            alt="Vuetify Logo"
            class="shrink mr-2"
            contain=""
            src="https://cdn.vuetifyjs.com/images/logos/vuetify-logo-dark.png"
            transition="scale-transition"
            width="40"
          />
           
          <v-img
            alt="Vuetify Name"
            class="shrink mt-1 hidden-sm-and-down"
            contain=""
            min-width="100"
            src="https://cdn.vuetifyjs.com/images/logos/vuetify-name-dark.png"
            width="100"
          />
        </div>
         
        <v-spacer />
         
        <v-btn
          href="https://github.com/vuetifyjs/vuetify/releases/latest"
          target="_blank"
          text=""
        >
          <span
            class="mr-2"
          >
            Latest Release
          </span>
           
          <v-icon>
            mdi-open-in-new
          </v-icon>
        </v-btn>
      </v-app-bar>
       
      <v-main>
        <nav>
          <a
            aria-current="page"
            class="router-link-exact-active router-link-active"
            href="#/"
          >
            Home
          </a>
           |
      
          <a
            class=""
            href="#/about"
          >
            About
          </a>
        </nav>
         
        <v-container>
          <v-row
            class="text-center"
          >
            <h1>
              
            </h1>
             
            <v-col
              cols="12"
            >
              <v-img
                class="my-3"
                contain=""
                height="200"
                src=""
              />
            </v-col>
             
            <v-col
              class="mb-4"
            >
              <h1
                class="display-2 font-weight-bold mb-3"
              >
                Welcome to Vuetify
              </h1>
               
              <p
                class="subheading font-weight-regular"
              >
                
        For help and collaboration with other Vuetify developers,
        
                <br />
                please join our online
        
                <a
                  href="https://community.vuetifyjs.com"
                  target="_blank"
                >
                  Discord Community
                </a>
              </p>
            </v-col>
             
            <v-col
              class="mb-5"
              cols="12"
            >
              <h2
                class="headline font-weight-bold mb-3"
              >
                What's next?
              </h2>
               
              <v-row
                justify="center"
              >
                <a
                  class="subheading mx-3"
                  href="https://vuetifyjs.com/components/api-explorer"
                  target="_blank"
                >
                  
          Explore components
        
                </a>
                <a
                  class="subheading mx-3"
                  href="https://vuetifyjs.com/getting-started/pre-made-layouts"
                  target="_blank"
                >
                  
          Select a layout
        
                </a>
                <a
                  class="subheading mx-3"
                  href="https://vuetifyjs.com/getting-started/frequently-asked-questions"
                  target="_blank"
                >
                  
          Frequently Asked Questions
        
                </a>
              </v-row>
            </v-col>
             
            <v-col
              class="mb-5"
              cols="12"
            >
              <h2
                class="headline font-weight-bold mb-3"
              >
                Important Links
              </h2>
               
              <v-row
                justify="center"
              >
                <a
                  class="subheading mx-3"
                  href="https://vuetifyjs.com"
                  target="_blank"
                >
                  
          Documentation
        
                </a>
                <a
                  class="subheading mx-3"
                  href="https://community.vuetifyjs.com"
                  target="_blank"
                >
                  
          Chat
        
                </a>
                <a
                  class="subheading mx-3"
                  href="https://madewithvuejs.com/vuetify"
                  target="_blank"
                >
                  
          Made with Vuetify
        
                </a>
                <a
                  class="subheading mx-3"
                  href="https://twitter.com/vuetifyjs"
                  target="_blank"
                >
                  
          Twitter
        
                </a>
                <a
                  class="subheading mx-3"
                  href="https://medium.com/vuetify"
                  target="_blank"
                >
                  
          Articles
        
                </a>
              </v-row>
            </v-col>
             
            <v-col
              class="mb-5"
              cols="12"
            >
              <h2
                class="headline font-weight-bold mb-3"
              >
                Ecosystem
              </h2>
               
              <v-row
                justify="center"
              >
                <a
                  class="subheading mx-3"
                  href="https://github.com/vuetifyjs/vuetify-loader"
                  target="_blank"
                >
                  
          vuetify-loader
        
                </a>
                <a
                  class="subheading mx-3"
                  href="https://github.com/vuetifyjs/vuetify"
                  target="_blank"
                >
                  
          github
        
                </a>
                <a
                  class="subheading mx-3"
                  href="https://github.com/vuetifyjs/awesome-vuetify"
                  target="_blank"
                >
                  
          awesome-vuetify
        
                </a>
              </v-row>
            </v-col>
             
            <v-col
              class="mb-5"
              cols="12"
            >
              <h2>
                Call API when page mounted:
              </h2>
               
              <v-row
                justify="center"
              >
                <p>
                  {}
                </p>
              </v-row>
            </v-col>
          </v-row>
        </v-container>
      </v-main>
    </v-app>
  </v-container>
</v-app>
`;
